consensus/istanbul/core/core.go

```diff
@@ -781,6 +794,8 @@ func (c *core) verifyProposal(proposal istanbul.Proposal) (time.Duration, error)
 		return 0, verificationStatus
 	} else {
 		logger.Trace("verification status cache miss")
+		start := time.Now()
+		defer func() { c.verifyGauge.Update(time.Since(start).Nanoseconds()) }()
```

Comment:
Not much of a difference, but I think this (and the others like it in this PR) could be done as:
```
defer func(start time.Time) { c.verifyGauge.Update(time.Since(start).Nanoseconds()) }(time.Now())
```

e.g. https://play.golang.org/p/XzlpNvFellY